android/app/src/main/res/values-port/bools.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <bool name="isPortraitScreen">true</bool>
```

Comment:
Is this method not going to work?
https://stackoverflow.com/a/2799001/9710294

Reply:
That should work, will change.

Reply:
Done.